pkg/nuctl/test/project_test.go

```diff
@@ -38,38 +42,36 @@ type projectGetTestSuite struct {
 
 func (suite *projectGetTestSuite) TestGet() {
 	numOfProjects := 3
-	var projectNames []string
+	projectNames := make([]string, numOfProjects)
 
 	// get with nothing created - should pass
 	err := suite.ExecuteNuctl([]string{"get", "project"}, nil)
 	suite.Require().NoError(err)
 
 	for projectIdx := 0; projectIdx < numOfProjects; projectIdx++ {
-		uniqueSuffix := fmt.Sprintf("-%s-%d", xid.New().String(), projectIdx)
-
-		projectName := "get-test-project" + uniqueSuffix
+		projectName := fmt.Sprintf("get-test-project-%s-%d", xid.New().String(), projectIdx)
 
 		// add project name to list
-		projectNames = append(projectNames, projectName)
+		projectNames[projectIdx] = projectName
```

Comment:
this + line 45  - just change for the sake of change
Initializing a slice with size 3 and setting by index instead of appending achieves nothing (optimization for slice length of 3) over the previous implementation

Reply:
well, I've changed it because - hint errors, `array not being initialized` message was pretty much annoying.